### PR TITLE
Update pass on docs.begin.com

### PR DESCRIPTION
**Pull request by macdonst**
I'm doing a pass on docs.begin.com updating docs that are a bit out of date.

### CODE REVIEW

File: src/shared/docs/en/guides/quickstart.md [hunk 1]
```diff
@@ -17,27 +17,27 @@ While this guide assumes general familiarity with such things as text editors, t
...
 ### Platforms: macOS, Windows, and Linux
 
 Begin supports local, offline development on just about all modern computers, including: macOS, Windows (10+), and Linux.
 
 
-### Runtimes: Node.js 12
+### Runtimes: Node.js
 
-Begin provisions and manages AWS infrastructure that runs solely on Node.js 12 (with support additional runtimes coming soon).
+Begin provisions and manages AWS infrastructure that runs on Node.js (with support for additional runtimes coming soon). Node.js 12.x is the default runtime, but you can [configure](https://arc.codes/docs/en/reference/project-manifest/aws#runtime) Node.js 14.x to be the runtime on a per-project or per-function level.
```
Comment (ryanbethel) [inline src/shared/docs/en/guides/quickstart.md:31]: I think that node 14 is now the default.

**Review by ryanbethel — COMMENTED** [review]
Looks good. As I mentioned in the other comment I think node 14 is now the default if not specified.

File: src/shared/docs/en/guides/quickstart.md (hunk 1 above)
Comment (macdonst) [reply src/shared/docs/en/guides/quickstart.md:31]: @ryanbethel can you try deploying a new app and checking the node version number? It seems to me that it is still node 12.

File: src/shared/docs/en/guides/quickstart.md [hunk 2]
```diff
@@ -17,27 +17,27 @@ While this guide assumes general familiarity with such things as text editors, t
 ### tl;dr, you'll need:
 - A (free) GitHub account
 - Any computer running macOS, Windows, or Linux, with:
-  - Node.js 12
-  - npm 6.x
+  - Node.js 12, or later
```
Comment (tbeseda) [inline src/shared/docs/en/guides/quickstart.md:20]: I believe Node 14 is required for Sandbox now. Lambdas still support Node 12, but running Sandbox needs Node 14+

**Review by tbeseda — CHANGES_REQUESTED** [review]

File: src/shared/docs/en/guides/quickstart.md (hunk 2 above)
Comment (macdonst) [reply src/shared/docs/en/guides/quickstart.md:20]: Yes, good clarification. I will make that change.

**Review by tbeseda — APPROVED** [review]
